gothenburg/meetup.yaml

```diff
@@ -0,0 +1,77 @@
+city: Göteborg
+country: sweden
+meetupID: Kubernetes-Goteborg
+meetups:
+- address: Kungsportsavenyen 33
+  attendees: 45
+  date: "2019-05-09T18:00:00Z"
+  duration: 3h0m0s
+  id: 260863253
+  name: 18 months of kubernetes in production and learning
+  presentations:
+  - duration: 30m0s
+    slides: ""
+    speakers:
+    - jessicaandersson
+    title: Getting Developers to Adopt Your Service
+  - delay: 5m0s
+    duration: 30m0s
+    slides: ""
+    speakers:
+    - federicohernandez
+    title: Federico Hernandez - Knowing what your Kubernetes cluster is doing
+  - delay: 5m0s
+    duration: 10m0s
+    slides: ""
+    speakers:
+    - simonesciarrati
+    title: Simone Sciarrati - Zero downtime upgrades of Kubernetes
+  sponsors:
+    other: null
```

Comment:
add CNCF?

Reply:
well, not really yet :) For next meetup, sure!